docs/generated.d.ts

```diff
@@ -3059,6 +3063,40 @@ export interface TxSearchResult {
     metadata?: Transaction;
   };
 }
+/**
+ * GET request that returns the deployment status of multiple smart contracts
+ */
+export interface SmartContractsStatusResponse {
+  [k: string]: (SmartContractFound | SmartContractNotFound) | undefined;
+}
+export interface SmartContractFound {
+  found: true;
+  status?: SmartContractStatus;
+}
+/**
+ * Deployment status of a smart contract
+ */
+export interface SmartContractStatus {
+  /**
+   * Smart contract deployment transaction status
+   */
+  status: string;
+  /**
+   * Deployment transaction ID
+   */
+  tx_id: string;
```

Comment:
Just curious, how hard is it to get the tx_status added to this call? This would mean needing to one less call to go from contract txid to requesting the tx list endpoint for tx status

Reply:
`status` is actually the `tx_status`, perhaps I should rename it